.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -54,7 +54,7 @@ env:
   # renovate: datasource=github-releases depName=kubernetes-sigs/kind
   kind_version: v0.20.0
   # renovate: datasource=docker depName=kindest/node
```

Comment:
Can we replace with `# renovate: datasource=docker depName=cilium/kindest-node`?